src/components/RateAction/RateAction.css.js

```diff
@@ -0,0 +1,186 @@
+import styled from '../styled'
+import baseStyles from '../../styles/resets/baseStyles.css'
+import { getColor } from '../../styles/utilities/color'
+import { reactionEmoticonsColours } from '../Emoticon/Emoticon.css'
+
+export const config = {
+  size: {
+    default: '28px',
+    lg: '28px',
+    md: '24px',
+    sm: '20px',
+  },
+  boxShadowColor: '193, 203, 212', // grey.600
```

Comment:
there is a rgba function in the color utils. I think you could used that to keep rbg color out

it works like the sass method from what I recall `rgba(color, opacity)`